app/code/Magento/Sales/Model/AdminOrder/Create.php

```diff
@@ -232,6 +232,11 @@ class Create extends \Magento\Framework\DataObject implements \Magento\Checkout\
     private $serializer;
 
     /**
+     * @var \Magento\Framework\Locale\FormatInterface $localeFormat
+     */
+    protected $localeFormat;
```

Comment:
all new properties added should be private.
Inheritance based APIs are highly discouraged